feat: detect worst date

sgratzl opened the pull request:
closes #764 

**Prerequisites**:

- [x] Unless it is a hotfix it should be merged against the `dev` branch
- [x] Branch is up-to-date with the branch to be merged with, i.e. `dev`
- [x] Build is successful
- [x] Code is cleaned up and formatted

### Summary

![image](https://user-images.githubusercontent.com/4129778/109668738-bc8fe580-7b71-11eb-972d-1a45af3f6a90.png)

tildechris commented:
It looks like the best value comparison is always 0%.  "best value of 0.00% on Fri, Feb 14th."   Wait, is that from 2020?

sgratzl commented:
I changed it to the window trend (4 month) value. and added the year:

![image](https://user-images.githubusercontent.com/4129778/109681921-3b8b1b00-7b7e-11eb-817c-da3afedeb769.png)

tildechris reviewed (approved):
LGTM w/ a small fix: can we bold "worst"?  [Test url](https://deploy-preview-768--cmu-delphi-covidcast.netlify.app/indicator/?region=42003&date=20210210)

"On Thu, Feb 25th **COVID-Related Doctor Visits** was the 4 month **worst value** compared to **best value** of **3.56%** on Mon, Nov 2nd 2020."
